js/repl/ReplOptions.js

```diff
@@ -450,7 +450,7 @@ const styles = {
     }
   }),
   inputCheckboxLeft: css({
-    marginRight: '0.5rem'
+    margin: '0 0.5rem 0 0 !important' // Clashes with default input[type="checkbox"] styles in main.css
```

Comment:
> The problem is that we set (gross) global styles for checkboxes and radio.

Isn't that relatively common though, particularly for sites with custom-styled checkboxes (I'm not sure if Babel site has that)?

> I don't think more-specific would be any less hacky in this case,

I think it's less hacky than `!important`... Your intent is to use more specific styling for this particular checkbox, and that's what nested selectors are for in CSS 😛 

I guess either way is fine.